server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java

```diff
@@ -690,14 +700,14 @@ private void deduplicate(QueryPlan queryPlan) throws MetadataException {
 
     int index = 0;
     for (Pair<PartialPath, Integer> indexedPath : indexedPaths) {
-      String column;
-      if (indexedPath.left.getAlias() != null) {
-        column = indexedPath.left.getFullPathWithAlias();
-      } else {
-        column = indexedPath.left.getFullPath();
-      }
-      if (queryPlan instanceof AggregationPlan) {
-        column = queryPlan.getAggregations().get(indexedPath.right) + "(" + column + ")";
+      // judge whether as clause is used or not first
```

Comment:
what is not first?

Reply:
If as clause is used, measurementAlias will be covered.